libandroid-navigation-ui/src/test/java/com/mapbox/services/android/navigation/ui/v5/voice/SpeechAnnouncementTest.java

```diff
@@ -0,0 +1,40 @@
+package com.mapbox.services.android.navigation.ui.v5.voice;
+
+import com.mapbox.services.android.navigation.v5.milestone.VoiceInstructionMilestone;
+
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class SpeechAnnouncementTest {
```

Comment:
Test that AutoValue works as expected is not worth it IMO. Ultimately, it's like testing a setter. 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-droidcon-berlin-2016?slide=68